bin/cli.js

```diff
@@ -2,12 +2,31 @@
 
 'use strict';
 
+const yargs = require('yargs');
 const readline = require('readline');
 
 const Cli = require('../lib/cli/cli');
 
 let rl = null;
 
+const args = yargs
+  .option('verbose', {
+    alias: 'v',
+    count: true,
+    describe: 'Verbosely print incoming and outgoing messages',
+  })
+  .options('no-reconnect', {
```

Comment:
It is `options` method here and `option` methods for `'verbose'` and `'heartbeat-interval'`, is there a difference, or is this just a typo?

Reply:
After taking a look at `yargs` documentation, it seems that the `options` method has a different contract, requiring to provide the object with options descriptions as a single argument, meaning that the way you use that method here is incorrect.